### PR TITLE
ChangeTracking (Java): Add detail on Fiori version and event handler

### PR DESCRIPTION
**Pull request by vmikhailenko**
Adds one missing precondition (Fiori version needs to be updated)
Adds an example of the event handler.

### CODE REVIEW

File: java/change-tracking.md [hunk 1]
```diff
@@ -225,6 +227,30 @@ the root of the change is an order.
...
+## Reacting on Changes
+
+You can write the event handler to observe the change log entries. Keep in mind, that the change log entries 
+are created for each statement and this event will not be bound to any kind of transaction or a batch operation.
+
+```java
+import cds.gen.sap.changelog.Changes;
+
+@Component
+@ServiceName("ChangeTrackingService$Default")
```
Comment (BraunMatthias) [inline java/change-tracking.md:239]: We should make use of the constant for demo purpose here

File: java/change-tracking.md [hunk 2]
```diff
@@ -225,6 +227,30 @@ the root of the change is an order.
...
+public class ChangeTrackingHandler implements EventHandler {
+	
+  @After(event = "createChanges")
```
Comment (BraunMatthias) [inline java/change-tracking.md:242]: dito

File: java/change-tracking.md [hunk 3]
```diff
@@ -225,6 +227,30 @@ the root of the change is an order.
...
+  void afterCreate(EventContext context) {
+    Result result = (Result) context.get("result");
+	result.listOf(Changes.class).forEach(c -> {
```
Comment (BraunMatthias) [inline java/change-tracking.md:245]: indetation

File: java/change-tracking.md (hunk 1 above)
Comment (vmikhailenko) [reply java/change-tracking.md:240]: First, we need to expose them. They are not available when dependency is included with runtime scope.

**Comment by renejeglinsky** [comment]
@BraunMatthias Could you review and/or approve? Then we could include it in the release. Thanks :)

**Review by renejeglinsky — APPROVED** [review]

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**